<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddWindow</class>
 <widget class="QWidget" name="AddWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>574</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>571</width>
     <height>571</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="attribute">
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>340</y>
       <width>31</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;列：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>280</y>
       <width>31</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;行：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>220</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;资源一次增加量：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>100</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;初始资源：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>160</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;资源增长速度：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>50</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;生命值：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="health_box">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>60</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>999999</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="original_source_box">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>110</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maximum">
      <number>999999</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="speed_box">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>170</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>999999</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="colomn_box">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>350</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimum">
      <number>3</number>
     </property>
     <property name="maximum">
      <number>30</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="row_box">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>290</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimum">
      <number>3</number>
     </property>
     <property name="maximum">
      <number>30</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="add_once_box">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>230</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maximum">
      <number>999999</number>
     </property>
    </widget>
    <widget class="QPushButton" name="confirm_button">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>500</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>确认</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>400</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;路径条数：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="path_cnt_box">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>410</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="path">
    <widget class="QLabel" name="path_infor">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>231</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;设置路径&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="reset_button">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>500</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>重新选择</string>
     </property>
    </widget>
    <widget class="QPushButton" name="confirm_path">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>500</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>确认</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>20</y>
       <width>91</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;路径种类：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QComboBox" name="path_kind_box">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>40</y>
       <width>86</width>
       <height>25</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>地面</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>飞行</string>
      </property>
     </item>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
